lib/patch_ruby/models/project.rb

```diff
@@ -77,28 +77,6 @@ class Project
     # An array of objects containing available inventory for a project. Available inventory is grouped by a project's vintage year and returns amount and pricing available for a given vintage year.
     attr_accessor :inventory
 
-    class EnumAttributeValidator
```

Comment:
This happened because I removed the enum on the `mechanism` field in swagger (because it wasn't true anymore) if the value can be returned in different languages.